### PR TITLE
Fix less config with latest cra

### LINKED ISSUE #67: react-app-rewire-less not working
```
/Users/afc163/Projects/antd-demo-new/node_modules/_react-app-rewire-less@2.0.4@react-app-rewire-less/index.js:8
  fileLoader.exclude.push(lessExtension);
            ^

TypeError: Cannot read property 'exclude' of undefined
    at rewireLess (/Users/afc163/Projects/antd-demo-new/node_modules/_react-app-rewire-less@2.0.4@react-app-rewire-less/index.js:8:13)
    at override (/Users/afc163/Projects/antd-demo-new/config-overrides.js:6:12)
    at Object.<anonymous> (/Users/afc163/Projects/antd-demo-new/node_modules/_react-app-rewired@1.2.0@react-app-rewired/scripts/start.js:18:3)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:389:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:504:3
```

### PR DESCRIPTION
**Pull request by afc163**
close #67 

@timarney

### CODE REVIEW

**Comment by Guria** [comment]
In #71 I introduce better way to traverse rules. Hope to get it merged soon.

**Comment by timarney** [comment]
@afc163 please rework based on https://github.com/timarney/react-app-rewired/pull/71  and we'll get that merged in.

File: packages/react-app-rewired/index.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const babelLoaderMatcher = function(rule) {
-  return rule.loader && rule.loader.indexOf("/babel-loader/") != -1;
+  return rule.loader && rule.loader.indexOf("babel-loader/") != -1;
```
Comment (afc163) [inline packages/react-app-rewired/index.js]: The npminstall loader path would be like:

```
node_modules/_babel-loader@7.1.1@babel-loader/lib/index.js
```

**Comment by afc163** [comment]
Done!

File: packages/react-app-rewire-less/index.js [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
+  const fileLoader = getLoader(
+    config.module.rules, rule => rule.loader && rule.loader.endsWith(`file-loader${path.sep}index.js`)
+  );
   fileLoader.exclude.push(lessExtension);
 
-  const cssRules = config.module.rules
-    .find(rule => String(rule.test) === String(/\.css$/));
+  const cssRules = getLoader(
+    config.module.rules, rule => String(rule.test) === String(/\.css$/)
+  ;
```
Comment (afc163) [inline packages/react-app-rewire-less/index.js]: missing a `)` ..

Comment (afc163) [reply packages/react-app-rewire-less/index.js]: https://github.com/timarney/react-app-rewired/pull/73

**Pull request merged**